variants/frontend-react/app/frontend/tests/HelloWorld.spec.jsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import HelloWorld from '../components/HelloWorld';
+
+describe('HelloWorld', () => {
```

Comment:
nice! ❤️ 

This is a great test example - much better than what I came up with last time I played with testing library 😅